src/Core/tests/DeviceTests/Handlers/Label/LabelHandlerTests.cs

```diff
@@ -150,52 +150,7 @@ public async Task FontDoesNotAffectHorizontalTextAlignment(double initialSize, d
 				nameof(ILabel.Font),
 				() => label.Font = Font.SystemFontOfSize(newSize));
 		}
-#if !WINDOWS
```

Comment:
Moved these all into `TextStyleHandlerTests` since they apply to anything that implements `ITextStyle`